src/qforte/adapters/molecule_adapters.py

```diff
@@ -279,6 +282,25 @@ def create_psi_mol(**kwargs):
     # run psi4 caclulation
     p4_Escf, p4_wfn = psi4.energy('SCF', return_wfn=True)
 
+    # Get symmetry information
+    orbitals = []
+    for irrep in range(len(p4_wfn.epsilon_a().nph)):
+        for orbital in p4_wfn.epsilon_a().nph[irrep]:
```

Comment:
```suggestion
    for irrep in p4_wfn.epsilon_a().nph:
        for orbital in irrep:
```

Reply:
This is not correct. The way I am doing it creates a list with elements of the form `[orbital energy, orbital irrep index]`. This allows me to sort the orbital irreps according to the orbital energies. What you are suggesting creates a list with elements of the form `[orbital energy, numpy array of orbital energies of a given irrep]`, which is not what we need. I will stick with the original version of the code.